source/developer/bot-accounts.rst

```diff
@@ -33,36 +33,37 @@ If you would like to see improvements to bot accounts, `let us know in the Featu
   :local:
 
 Configuration Settings
-------------------------
+----------------------
 
 By default, plugins can create and manage bot accounts. To enable bot account creation through the user interface or the RESTful API:
 
 1. Go to **System Console > Integrations > Bot Accounts**.
 2. Set **Enable Bot Account Creation** to ``true``.
 
-Once set, System Administrators can create bot accounts for integrations using the **Integrations > Bot Accounts** link in the description provided.
+Once set, System Admin can create bot accounts for integrations using the **Integrations > Bot Accounts** link in the description provided.
 
 Bot Account Creation
------------------------
+--------------------
 
 Below are different ways to create bot accounts. After the bot account is created, make sure to:
 
 1. Copy the generated bot access token for your integration.
 2. Add the bot account to teams and channels you want it to interact in.
 
 User Interface (UI)
-^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^
 
 1. Go to **Main Menu > Integrations > Bot Accounts**.
-2. Click **Add Bot Account**.
-3. Set the **Username** of the bot. The username must begin with a letter, and contain between 3 and 22 lowercase characters made up of numbers, letters, and the symbols ".", "-", and "_".
-4. (Optional) Upload an image for the **Bot Icon**. This will be used as the profile image of the bot throughout the Mattermost user interface.
+2. Select **Add Bot Account**.
+3. Set the **Username** of the bot. The username must begin with a letter, and contain between 3 and 22 lowercase characters made up of numbers, letters, and symbols including ".", "-", or "_".
+4. (Optional) Upload an image for the **Bot Icon**. This will be used as the profile image of the bot throughout Mattermost.
```

Comment:
Is "Bot Icon" what displays in the UI? I ask because typically when referring to icons and UI options, we bold the name of the option itself, and leave the type of UI control unbolded and lowercase.

Reply:
**Bot Icon** is the name of the UI entity related to this action. It's a bit less clear-cut than usual:
<img width="357" alt="Screen Shot 2021-01-24 at 22 27 24" src="https://user-images.githubusercontent.com/4223530/105642598-5689b180-5e93-11eb-9ad2-a3d7187050a6.png">